src/testing/loadtest-azure/scripts/accesstoken.txt

```diff
@@ -0,0 +1 @@
+Authorization=Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6ImpTMVhvMU9XRGpfNTJ2YndHTmd2UU8yVnpNYyIsImtpZCI6ImpTMVhvMU9XRGpfNTJ2YndHTmd2UU8yVnpNYyJ9.eyJhdWQiOiJodHRwczovL2xvYWR0ZXN0LmF6dXJlLWRldi5jb20iLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC83MmY5ODhiZi04NmYxLTQxYWYtOTFhYi0yZDdjZDAxMWRiNDcvIiwiaWF0IjoxNjUxNzQ3ODI3LCJuYmYiOjE2NTE3NDc4MjcsImV4cCI6MTY1MTc1MjI2NywiX2NsYWltX25hbWVzIjp7Imdyb3VwcyI6InNyYzEifSwiX2NsYWltX3NvdXJjZXMiOnsic3JjMSI6eyJlbmRwb2ludCI6Imh0dHBzOi8vZ3JhcGgud2luZG93cy5uZXQvNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3L3VzZXJzLzk5OTIyYzFmLWZlMTAtNGQ5Ni05MzdjLTk3NTlkNmZhMDZmYy9nZXRNZW1iZXJPYmplY3RzIn19LCJhY3IiOiIxIiwiYWlvIjoiQVhRQWkvOFRBQUFBZVlTQ3RKS1ZxV0Z2Q1VyVmFsS0pCRmNpRmZUSFErcitTNE5RN3laZjlrWENKeTAxaWM1V2d4aTlKS2x5ZlhIM2luREZpZytCb1lBSlR2eHZ3ZXR1VVBEdS9nUVZ3VXBxSXN1SlJ3ZnViV1o0UHhyL2hLdWM1cnNGazRXWWtkUUhlNDdTR1Rxdk5kV09IQUd2NWhxUVp3PT0iLCJhbXIiOlsicnNhIiwibWZhIl0sImFwcGlkIjoiMDRiMDc3OTUtOGRkYi00NjFhLWJiZWUtMDJmOWUxYmY3YjQ2IiwiYXBwaWRhY3IiOiIwIiwiZGV2aWNlaWQiOiJkYTA4NGFkMi1hZjE1LTRjOTYtODc4NS1kN2I1ZTc2MjE1OTkiLCJmYW1pbHlfbmFtZSI6Ik9lbHJpY2hzIiwiZ2l2ZW5fbmFtZSI6IkhleWtvIiwiaXBhZGRyIjoiMjE3LjgwLjE1MS4xOCIsIm5hbWUiOiJIZXlrbyBPZWxyaWNocyIsIm9pZCI6Ijk5OTIyYzFmLWZlMTAtNGQ5Ni05MzdjLTk3NTlkNmZhMDZmYyIsIm9ucHJlbV9zaWQiOiJTLTEtNS0yMS0xNzIxMjU0NzYzLTQ2MjY5NTgwNi0xNTM4ODgyMjgxLTM3MzYwMDQiLCJwdWlkIjoiMTAwM0JGRkQ4RTUyRDZGQyIsInJoIjoiMC5BUm9BdjRqNWN2R0dyMEdScXkxODBCSGJSOEhVZTRTR2RFRkNwNFA1dmFhU1FjRWFBTFkuIiwic2NwIjoidXNlcl9pbXBlcnNvbmF0aW9uIiwic3ViIjoiSjBNRmZyV0V1XzFta3d0OXNOUlRLX2xRQ0FkNnl2bzE0cFZuallFbGYxayIsInRpZCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsInVuaXF1ZV9uYW1lIjoiaGVvZWxyaUBtaWNyb3NvZnQuY29tIiwidXBuIjoiaGVvZWxyaUBtaWNyb3NvZnQuY29tIiwidXRpIjoibXFqc3U2aVR2MHFPQUlkcjFtSF9BQSIsInZlciI6IjEuMCIsIndpZHMiOlsiYjc5ZmJmNGQtM2VmOS00Njg5LTgxNDMtNzZiMTk0ZTg1NTA5Il19.X9WumhY0A6XK1Z7uC4VFtl4rCw9HHjOBIfrpEJxpaJHIj8kEJ-tm6rsmSqrV9xww_D1u0HdpUdEE1jemsElS-BP1tcz9qqU49fI5H7HQu4H-oysof5hQPpfEknkuHKejVDqBfoUQRd-fUZIl22urYSOI8T8jHoMF-erklH1ahQLU7H20pW8LUpm-rDX4AYtzNfmot7VbvFrLuUIH7H2ZBt_C_P4LcSqKgDzdG3FeP7B-OnIReYbKjbzHflD2PWxHb97jiWicLcZyopPP_Y5cMofalqTq3ItrPGxKDpwqIMFBSWsHMBvWdL9yy3HjaV0pLJO8s_q2ckdLUcDgoh1owg
```

Comment:
Storing access tokens in a repo (especially an open source one) is normally a bad idea. What's the purpose of this?

Reply:
You're of course right - this file shouldn't be there. Seems like it somehow fall out of the `.gitignore` file.